platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu-platform-init.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Pensando DPU platform init service
+Requires=dpu.service
+
+BindsTo=sonic.target
+After=sonic.target
+
+[Service]
+Type=oneshot
+ExecStartPre=/bin/sleep 15
```

Comment:
Please remove sleep here if it's not needed. If it's needed, please determine what it is actually waiting on and add a check for that instead.

Reply:
done